### PR TITLE
Actionmenu percept rules added

### PR DESCRIPTION
**Pull request by hmyeh**
Added action_menu percept to the GOAL bot.

As a goal bot I want to  be able to see the new action_menu percept added to my belief base so that I can use it to decide on which action to perform.

### CODE REVIEW

**Comment by dionytadema** [comment]
:+1:

**Comment by Stefan1de1V** [comment]
I think the init has the same lines twice? Unless my eyes are deceiving me.

**Comment by levilime** [comment]
It looks good, but I'm not sure if it is useful yet, I don't agree with the merge for now, because I'm not sure if we have use for it yet.

**Comment by NickTUD** [comment]
Again has merge conflicts btw

File: Municipality/tygronInit.mod2g [hunk 1]
```diff
@@ -49,6 +49,17 @@ module tygronInit {
 		forall percept(requests(List)), bel(member(request(IDType,AnswerList),List))
 			do insert(request(IDType,AnswerList)).
 		
+		%%% ACTION PERCEPT
+		%%% Action insert rule, since the action percepts already has actions in it during the first cycle.
```
Comment (NickTUD) [inline Municipality/tygronInit.mod2g]: Make comments uniform -> change %%% to % and remove the space after %

**Comment by levilime** [comment]
I will close this, because it looks like we won't be needing the logic for action menu.

**Pull request closed without merging**